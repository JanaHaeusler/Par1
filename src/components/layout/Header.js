import styled from 'styled-components/macro'
import { Logo } from '../Icons'

export default function Header() {

  return (
    <HeaderStyled>
      <PennantStyled>
        <LogoStyled/>
      </PennantStyled>
    </HeaderStyled>
  )
}

const HeaderStyled = styled.header`
  height: 50px;
  box-shadow: 0 0 10px var(--primary-dark);
  border-radius: 0 0 25px 3px;
  background: var(--secondary-dark);
`
const PennantStyled = styled.div`
  position: absolute;
  top: 0;
  left: 28px;
  width: 60px;
  height: 60px;
  display: flex; 
  justify-content: center;
  box-shadow: 0 0 10px var(--primary-dark);
  border-radius: 0 0 50px 50px;
  background-color: var(--text-light);
`
const LogoStyled = styled(Logo)`
  margin-top: 10px;
  width: 30px;
  fill: var(--secondary-dark);
`