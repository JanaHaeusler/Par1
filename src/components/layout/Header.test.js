import { render } from '@testing-library/react'
import Header from './Header'

describe('Header', () => {
    it('reders correctly', () => {
        const { container } = render(<Header/>)
        expect(container.firstChild).toMatchSnapshot()
    })
})