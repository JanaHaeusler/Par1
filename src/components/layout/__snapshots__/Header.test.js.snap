// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header reders correctly 1`] = `
.c0 {
  height: 50px;
  box-shadow: 0 0 10px var(--primary-dark);
  border-radius: 0 0 25px 3px;
  background: var(--secondary-dark);
}

.c1 {
  position: absolute;
  top: 0;
  left: 28px;
  width: 60px;
  height: 60px;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  box-shadow: 0 0 10px var(--primary-dark);
  border-radius: 0 0 50px 50px;
  background-color: var(--text-light);
}

.c2 {
  margin-top: 10px;
  width: 30px;
  fill: var(--secondary-dark);
}

<header
  class="c0"
>
  <div
    class="c1"
  >
    <svg
      class="c2"
    >
      golf-logo.svg
    </svg>
  </div>
</header>
`;
